tests/models_tests/models_test.go

```diff
@@ -188,15 +208,27 @@ func TestRetrieveBreakdownStatsBySourceIdAndCohortIdAndConceptIdsNoResults(t *te
 func TestRetrieveBreakdownStatsBySourceIdAndCohortIdAndConceptIdsWithResults(t *testing.T) {
 	setUp(t)
 	filterIds := make([]int64, 1)
-	filterIds[0] = genderConceptId
-	breakdownConceptId := genderConceptId // not normally the case...but we'll use the same here just for the test...
+	filterIds[0] = hareConceptId
+	breakdownConceptId := hareConceptId // not normally the case...but we'll use the same here just for the test...
 	stats, _ := conceptModel.RetrieveBreakdownStatsBySourceIdAndCohortIdAndConceptIds(testSourceId,
 		largestCohort.Id,
 		filterIds, breakdownConceptId)
-	// we expect values since all of the test cohorts have at least one subject with gender info:
-	if len(stats) < 2 {
+	// we expect values since largestCohort has multiple subjects with hare info:
+	if len(stats) < 4 {
 		t.Errorf("Expected at least two results, found %d", len(stats))
```

Comment:
should we change this to be at least four results?

Reply:
good catch